src/main/kotlin/solve/scene/model/VisualizationFrame.kt

```diff
@@ -6,7 +6,7 @@ import javafx.scene.image.Image
  * Frame data object.
  */
 data class VisualizationFrame(
-    val timestamp: Long,
+    val timestamp: Double,
```

Comment:
When I copy timestamp from the frame and it was initially integer 1 I have 1.0 in the clipboard